server/src/main/models/index.ts

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var fs: any        = require('fs');
```

Comment:
Why not `import * as fs from "fs"` and so on?

Reply:
This was the way it was generated but I will change it.